SecretSanta/src/SecretSanta.Data/DeleteMe.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace SecretSanta.Data{
+    public static class DeleteMe{
+        public static List<User> users {get;} = new (){
```

Comment:
Nit: Properties should be named with PascalCase
```suggestion
        public static List<User> Users { get; } = new (){
```